src/components/base/outline-alert/outline-alert.stories.ts

```diff
@@ -0,0 +1,158 @@
+import { html, TemplateResult } from 'lit';
+import './outline-alert';
+import { argTypeSlotContent } from '../../base/outline-element/utils/utils';
+import { alertSizes, alertStatusTypes } from './outline-alert';
+import { ifDefined } from 'lit/directives/if-defined';
+
+export default {
+  title: 'Molecules/Alert',
+  component: 'outline-alert',
+  argTypes: {
+    headerSlot: {
+      name: 'slot="outline-alert--header"',
+      description: 'The header slot. For example a header title.',
+      table: {
+        category: 'Slots',
+      },
+      // We aren't setting a control type here so we can edit the value using the infered object.
+    },
+    // We aren't setting a control type here so we can edit the value using the infered object.
+    defaultSlot: {
+      ...argTypeSlotContent.defaultSlot,
+      table: {
+        category: 'Slots',
+      },
+    },
+    statusType: {
+      description:
+        'The status type of the alert. Such as `information` or `warning`.',
+      options: alertStatusTypes,
+      control: { type: 'select' },
+    },
+    size: {
+      description: 'The size of the alert.',
+      options: alertSizes,
```

Comment:
@LeeMellon here is where the const assertion mentioned below comes in handy, now if you need to add a new alert size or modify an existing one, you just add it in the one file and you're good to go.